python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -280,6 +281,16 @@ def parse_arg(argv):
         action="store_true",
         help="show detailed joblog information, use with `--id`"
     )
+    ins_job.add_argument(
+        "--conf-url",
+        default="https://raw.githubusercontent.com/4paradigm/OpenMLDB/main/python/openmldb_tool/diagnostic_tool/common_err.yml",
```

Comment:
here and `CONF_URL`, two default value. Just use one. You can add openmldb.ai in help, and default use github source